src/FakeItEasy/ISequentialCallContext.cs

```diff
@@ -0,0 +1,23 @@
+namespace FakeItEasy
+{
+    using System;
+    using System.ComponentModel;
+    using FakeItEasy.Core;
+
+    /// <summary>
+    /// Represents a context to make assertions on the order of the calls made on a fake.
+    /// </summary>
+    public interface ISequentialCallContext : IHideObjectMembers
+    {
+        /// <summary>
+        /// This method supports the FakeItEasy infrastructure and is not intended to be used directly from your code.

```

Comment:
Even as I made the comment, I worried that people wouldn't see the remark. You've convinced me.